apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -100,23 +106,31 @@ contract DisputableVoting is IForwarderWithContext, DisputableAragonApp {
     struct Vote {
         uint256 yea;                                        // Voting power for
         uint256 nay;                                        // Voting power against
+        // Note: rename to totalPower?
         uint256 votingPower;                                // Total voting power (based on the snapshot block)
         uint256 settingId;                                  // Identification number of the setting applicable to the vote
         uint256 actionId;                                   // Identification number of the associated disputable action on the attached Agreement
+        // Note: we may want to re-order this state to be based on localized SSTOREs
+        //       start date and snapshot block never change, whereas pausedAt, pauseDuration and status may be changed together
         VoteStatus status;                                  // Status of the vote
         uint64 startDate;                                   // Datetime when the vote was created
         uint64 snapshotBlock;                               // Block number used to check voting power on attached token
         uint64 pausedAt;                                    // Datetime when the vote was paused
         uint64 pauseDuration;                               // Duration of the pause (only updated once resumed)
+        // Note: maybe rename to quietEndingExtensionDuration?
         uint64 quietEndingExtendedSeconds;                  // Total number of seconds a vote was extended due to quiet ending
+        // Note: I initially found this name a bit confusing, but it makes sense now
         VoterState quietEndingSnapshotSupport;              // Snapshot of the vote's support at the beginning of the first quiet ending period
+        // Note: let's prefer to store a hash here
         bytes executionScript;                              // EVM script attached to the vote
```

Comment:
I think this will save a lot of gas, which I think is much more important now that it can get quite expensive to create an action.

Reply:
Addressed in d3463b54a6d1039ddf15aae2e0ebc4eb7646bbfc